conan/api/model/list.py

```diff
@@ -65,22 +73,30 @@ def load_graph(graphfile, graph_recipes=None, graph_binaries=None, context=None)
         try:
             base_context = context.split("-")[0] if context else None
             graph = json.loads(load(graphfile))
-            mpkglist =  MultiPackagesList._define_graph(graph, graph_recipes, graph_binaries,
-                                                        context=base_context)
-            if context == "build-only":
-                host = MultiPackagesList._define_graph(graph, graph_recipes, graph_binaries,
-                                                       context="host")
-                mpkglist.keep_outer(host)
-            elif context == "host-only":
-                build = MultiPackagesList._define_graph(graph, graph_recipes, graph_binaries,
-                                                        context="build")
-                mpkglist.keep_outer(build)
-            return mpkglist
         except JSONDecodeError as e:
             raise ConanException(f"Graph file invalid JSON: {graphfile}\n{e}")
         except Exception as e:
             raise ConanException(f"Graph file broken: {graphfile}\n{e}")
 
+        if "graph" not in graph:
+            raise ConanException(
+                'Expected a graph file but found an unexpected JSON file format. You can create a "graph" JSON file by running:\n\n'
+                f"\tconan [ graph-info | create | export-pkg | install | test ] --format=json > graph.json\n\n"
+                "More Info at 'https://docs.conan.io/2/reference/commands/formatters/graph_info_json_formatter.html"
+            )
+
+        mpkglist =  MultiPackagesList._define_graph(graph, graph_recipes, graph_binaries,
+                                                    context=base_context)
+        if context == "build-only":
+            host = MultiPackagesList._define_graph(graph, graph_recipes, graph_binaries,
+                                                   context="host")
+            mpkglist.keep_outer(host)
+        elif context == "host-only":
+            build = MultiPackagesList._define_graph(graph, graph_recipes, graph_binaries,
+                                                    context="build")
+            mpkglist.keep_outer(build)
```

Comment:
If there is any other unexpected error in the graph.json, being it valid json, but fails to serialize, this will raise. It should keep the ``except Exception``, so maybe leave it in the original location and just add the new validation.